python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -224,18 +235,29 @@ def run(self, inputExp, sourceExps=[]):
                     if sourceAmpName == targetAmpName and sourceChip == targetChip:
                         ratioDict[targetAmpName][sourceAmpName] = []
                         continue
+
                     self.log.debug("    Target amplifier: %s", targetAmpName)
 
-                    targetAmpImage = CrosstalkCalib.extractAmp(targetIm.image,
+                    targetAmpImage = CrosstalkCalib.extractAmp(targetIm,
```

Comment:
I don't see any changes to the `extractAmp` signature, so I don't follow how it is now able to take `targetIm` instead of `targetIm.image`.

Reply:
Okay, I see that it can take either an `Image` instance or a `MaskedImage` instance and returns an object of the same type(?) I don't see why that change was necessary though.

Reply:
I think I wanted to look at the mask planes while I was debugging things.